src/adapters/scene_direction.js

```diff
@@ -30,10 +30,17 @@ const createRouteLabel = (route, vehicle, { lngLat, anchor }) => {
     <div class="routeLabel-vehicleIcon ${getVehicleIcon(vehicle)}"></div>
     <div>
       <div class="routeLabel-duration">${formatDuration(route.duration)}</div>
-      <div class="routeLabel-distance">${formatDistance(route.distance)}</div>
+      ${
```

Comment:
I know in React it's common not to render stuff in the markup, but here it's less pratical.
Maybe instead of changing the markup we could just use a semantic class name on the element, like `routeLabel--publicTransport` instead of `--single-line`, and use it to hide the distance in CSS.